keyboards/dp3000/config.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2023 deddia permana (@depermana12)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define LAYER_STATE_8BIT
```

Comment:
Just a heads up, if via is enabled, this will be automatically configured to the 8bit bitmask if there are 8 or fewer layers enabled for the dynamic keymap (default is 4).  So you may not need this here.

Reply:
Thank you, your explanation is very helpful